sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -32,6 +32,19 @@ dotnet add package Azure.Monitor.Query
 
 An authenticated client is required to query Logs or Metrics. To authenticate, create an instance of a [TokenCredential](https://docs.microsoft.com/dotnet/api/azure.core.tokencredential?view=azure-dotnet) class. Pass it to the constructor of your `LogsQueryClient` or `MetricsQueryClient` class.
 
+To authenticate, the following example uses `DefaultAzureCredential` from the [Azure.Identity](https://www.nuget.org/packages/Azure.Identity) package:
+
+```csharp
```

Comment:
Can you, please, use a snippet injection syntax? https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets